src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/EvidenceUploadHandlerTest.java

```diff
@@ -1,76 +1,246 @@
+import uk.gov.hmcts.reform.civil.service.Time;
 
-import java.util.HashMap;
-import java.util.Map;
-
+import static com.google.common.collect.Lists.newArrayList;
 import static java.util.Map.entry;
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
 import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_START;
 import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.MID;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.SUBMITTED;
 import static uk.gov.hmcts.reform.civil.callback.CaseEvent.EVIDENCE_UPLOAD;
+import static uk.gov.hmcts.reform.civil.utils.ElementUtils.element;
 
 @ExtendWith(SpringExtension.class)
 @SpringBootTest(classes = {
     EvidenceUploadHandler.class,
     JacksonAutoConfiguration.class
 })
-public class EvidenceUploadHandlerTest extends BaseCallbackHandlerTest {
+class EvidenceUploadHandlerTest extends BaseCallbackHandlerTest {
 
     @Autowired
     private EvidenceUploadHandler handler;
 
+    @MockBean
+    private Time time;
+
+    @Autowired
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    private final UploadEvidenceExpert uploadEvidenceDate = new UploadEvidenceExpert();
+    private final UploadEvidenceWitness uploadEvidenceDate2 = new UploadEvidenceWitness();
+
+    private static final String PAGE_ID = "validateValues";
+
+    @BeforeEach
+    void setup() {
+        given(time.now()).willReturn(LocalDateTime.now());
+    }
+
     @Test
     void givenAboutToStartThenReturnsAboutToStartOrSubmitCallbackResponse() {
+        // Given
         CaseData caseData = CaseDataBuilder.builder().build();
         CallbackParams params = callbackParamsOf(caseData, ABOUT_TO_START);
 
+        // When
         CallbackResponse response = handler.handle(params);
 
+        // Then
         assertThat(response).isInstanceOf(AboutToStartOrSubmitCallbackResponse.class);
     }
 
     @Test
-    void givenAboutToSubmitThenReturnsAboutToStartOrSubmitCallbackResponse() {
-        CaseData caseData = CaseDataBuilder.builder().build();
-        CallbackParams params = callbackParamsOf(caseData, ABOUT_TO_SUBMIT);
+    void shouldSucceed_whenExpertOption1UploadDatePast() {
+        // Given
+        List<Element<UploadEvidenceExpert>> date = newArrayList();
+        date.add(0, element(uploadEvidenceDate.toBuilder()
+                                .expertOption1UploadDate(time.now().toLocalDate().minusWeeks(1)).build()));
+
+        CaseData caseData = CaseDataBuilder.builder().atStateNotificationAcknowledged().build().toBuilder()
+            .documentUploadExpert1(date)
+            .build();
+        CallbackParams params = callbackParamsOf(caseData, MID, PAGE_ID);
+
+        // When
+        var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+
+        // Then
+        assertThat(response.getErrors()).isEmpty();
+    }
```

Comment:
Using the approach used in `shouldReturnError_whenExpertOptionUploadDateFuture`, make this test run against both `expertOption1UploadDate` and `expertOption2UploadDate`.